website/blog/authors.yml

```
@@ -92,3 +92,9 @@ vincentriemer:
   title: Software Engineer at Meta
   url: https://twitter.com/vincentriemer
   image_url: https://github.com/vincentriemer.png
+
+NickGerleman:
+  name: Nick Gerleman
+  title: Software Engineer at Meta
+  url: https://twitter.com/nick_gerleman
```

```suggestion
  url: https://github.com/NickGerleman
```
I don't really check Twitter, but it looks like there are some other author URLs using GitHub.